Detectors/TRD/calibration/include/TRDCalibration/PadCalibCCDBBuilder.h

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file PadCalibCCDBBuilder.h
+/// \brief Krypton calibration
+/// \author Jana Crkovska
+
+#ifndef O2_TRD_KRCALIBRATION_H
+#define O2_TRD_KRCALIBRATION_H
+
+#include "DataFormatsTRD/Constants.h"
+#include "Rtypes.h"
+#include "TH2.h"
+#include "TTree.h"
+#include <cstdlib>
+#include <numeric>
+#include <vector>
+
+namespace o2
+{
+namespace trd
+{
+
+class PadCalibCCDBBuilder
+{
+ public:
+  PadCalibCCDBBuilder(){};
+  ~PadCalibCCDBBuilder(){};
```

Comment:
default constructor and destructor should be marked with `= default` instead of  an empty block `{}`

Reply:
ok